source/Octopus.Tentacle/Kubernetes/KubernetesRawScriptPodCreator.cs

```diff
@@ -39,6 +40,7 @@ protected override async Task<IList<V1Container>> CreateInitContainers(StartKube
             {
                 Name = $"{podName}-init",
                 Image = command.PodImageConfiguration?.Image ?? await containerResolver.GetContainerImageForCluster(),
+                ImagePullPolicy = KubernetesConfig.ScriptPodPullPolicy.IsNullOrEmpty() ? null : KubernetesConfig.ScriptPodPullPolicy,
```

Comment:
Can this `.IsNullOrEmpty()` logic be moved into the KubernetesConfig?

Reply:
I actually suspect we just remove the ternary - whatever got passed in, goes here - it'll already be null.